src/pages/projects/EditProject.tsx

```diff
@@ -38,6 +39,10 @@ const EditProject: FC<Props> = ({ project }) => {
   const { hasProjectsNetworksZones, hasStorageBuckets } =
     useSupportedFeatures();
 
+  const { data: profile } = useQuery({
+    queryKey: [queryKeys.profiles, "default", "default"],
+    queryFn: () => fetchProfile("default", "default"),
```

Comment:
```suggestion
    queryKey: [queryKeys.profiles, "default", project.name],
    queryFn: () => fetchProfile("default", project.name),
```
We need the profile from the project we are editing here.